tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -239,3 +252,25 @@ Verify Errors In Jupyterhub Logs
         ...    container=${pod['spec']['containers'][0]['name']}
         Should Not Contain    ${logs}    ModuleNotFoundError: No module named 'distutils.util'
     END
+
+Verify Grafana Can Obtain Data From Prometheus Datasource
+    [Documentation]    Verifies Grafana can obtain data from Prometheus datasource
+    Login To Grafana
+    Click Element    xpath=/html/body/grafana-app/sidemenu/div[2]/div[3]/a/span
+    Wait Until Page Contains Element    xpath=//div[contains(@class,'grafana-select-value')]
+    Input Text    xpath=//div[contains(@class,'grafana-select-value')]//input    text=Monitoring
+    Press Keys    None    RETURN
+    Sleep    2    reason=Wait for Monitoring to get selected
+    Press Keys    xpath=//span[contains(text(),'Enter a PromQL query')]    traefik_backend_server_up
+    Press Keys    None    SHIFT+ENTER
+    Page Should Contain  text=Graph
+
+Login To Grafana
+    [Documentation]    Login to Grafana and wait until home page is loaded
+    ${grafana_url} =    Run    oc get routes/grafana -n redhat-ods-monitoring -o json | jq -r '.spec.host'
+    Open Browser    url=https://${grafana_url}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Wait Until Page Contains    text=Log in with OpenShift    timeout=15s
+    Login To RHODS Dashboard    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}    ${OCP_ADMIN_USER.AUTH_TYPE}
+    ${authorization_required} =    Is Service Account Authorization Required
+    Run Keyword If    ${authorization_required}    Authorize jupyterhub service account
```

Comment:
not sure this is the most suitable approach to implement grafana login because:
1. `Login To RHODS Dashboard` is checking for rhods service account authorization. Is this page appearing in grafana too?
2. line 275: the kw is checking for "jupyterhub" service account. I would expect something related to grafana

Maybe you can follow the same approach used to implement `Login To RHODS Dashboard` while writing the Grafana login kw

Reply:
Updated.